### PR TITLE
feat: add create alter table expr translator

### PR DESCRIPTION
**Pull request by WenyXu**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

1. Add alter table expr translator
2. Add create table expr translator

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.
- [x]  This PR does not require documentation updates.

## Refer to a related PR or issue link (optional)
#3174

### CODE REVIEW

**Review by zhongzc — APPROVED** [review]
🚀🚀🚀

File: tests-fuzz/tests/main.rs [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+};
+use tests_fuzz::generator::create_expr::CreateTableExprGeneratorBuilder;
+use tests_fuzz::generator::Generator;
+use tests_fuzz::translator::greptime::create_expr::CreateTableExprTranslator;
+use tests_fuzz::translator::DslTranslator;
+
+const GT_STANDALONE_MYSQL_ADDR: &str = "GT_STANDALONE_MYSQL_ADDR";
+const GT_STANDALONE_POSTGRES_ADDR: &str = "GT_STANDALONE_POSTGRES_ADDR";
+
+#[tokio::test]
```
Comment (waynexia) [inline tests-fuzz/tests/main.rs:33]: Not sure if it should be run as a unit test. Let take a look afterward

**Review by waynexia — APPROVED** [review]
🚀

File: tests-fuzz/tests/main.rs (hunk 1 above)
Comment (WenyXu) [reply tests-fuzz/tests/main.rs:33]: Let me remove it

**Pull request merged**